packages/common/src/interfaces.ts

```diff
@@ -258,9 +272,10 @@ export interface IItemTemplateConversions {
     templateDictionary: any,
     destinationAuthentication: UserSession,
     progressTickCallback: () => void
-  ): Promise<string>;
-  postProcessCircularDependencies?(
-    newItemTemplate: IItemTemplate,
+  ): Promise<ICreateItemFromTemplateResponse>;
+  postProcessDependencies?(
+    newItemTemplate: IItemTemplate[],
```

Comment:
Let's please rename the `newItemTemplate` array to `templates` to match function signature, and `authentication` is clearer than the function's `destinationAuthentication`

Reply:
good catch..will do